packages/console/src/components/Project/ProjectDashboard.tsx

```diff
@@ -166,7 +186,7 @@ export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({
       );
       return projectDomainAtributes;
     },
-    staleTime: Infinity,
+    enabled: !!projectAttributesQuery.data,
```

Comment:
The behaviour when the project settings error out is not clear. So we fetch projectDomainAttributes only once we have the upper level query.